doc/cephfs/mds-config-ref.rst

```diff
@@ -318,11 +300,26 @@
 
 ``mds bal fragment interval``
 
-:Description: The frequency (in seconds) of adjusting directory fragmentation.
+:Description: The delay (in seconds) between a fragment being elegible for split
+              or merge and executing the fragmentation change.
 :Type:  32-bit Integer
 :Default: ``5``
 
 
+``mds bal fragment fast limit``
+
+:Description: The ratio by which frags may exceed the split size before
+              a split is executed immediately (skipping the fragment interval)
+:Type:  Float
+:Default: ``1.5``
+
+``mds bal fragment size max``
+
+:Description: The maximum size of a fragment before any new entries
+              are rejected with ENOSPC.
+:Type:  32-bit Integer
+:Default: ``100000``
+
```

Comment:
It's in there, just awkward to grep because of interchangeable underscores vs spaces in config names